torch/utils/data/dataset.py

```diff
@@ -32,18 +32,27 @@ class TensorDataset(Dataset):
     Arguments:
         data_tensor (Tensor): contains sample data.
         target_tensor (Tensor): contains sample targets (labels).
+        *tensors (Tensor): tensors with the same first dimension.
     """
 
-    def __init__(self, data_tensor, target_tensor):
-        assert data_tensor.size(0) == target_tensor.size(0)
-        self.data_tensor = data_tensor
-        self.target_tensor = target_tensor
+    def __init__(self, data_tensor=None, target_tensor=None, *tensors):
```

Comment:
I don't think anyone has ever used those kwargs. Let's simplify the signature and take in `*tensors`.

Reply:
That would make it simpler.